pages/user guides/discovery_techniques.md

```diff
@@ -14,7 +14,7 @@ elements to the cloud. However, the path to the cloud can be long and painful
 without proper planning and execution.  By following Tidal Migration's six
 discovery layering techniques, you will be migrating to the cloud with ease!
 
-{% include image.html file="layered_discovery.png" caption="Tidal Migrations'
+{% include image.html file="layered_discovery_expanded.png" caption="Tidal Migrations'
```

Comment:
I think this change should be made once the feature is out of beta. ie. not necessarily this MR. 

What do you think?

Reply:
yes. It makes sense to me. I must have gotten confused about it and thought we wanted to push this right now. I'll leave the image in here but revert the file name back to the original. When it is time to make the change anyone can adjust this line.